coinmarketcap/src/endpoint/dominance.ts

```diff
@@ -0,0 +1,39 @@
+import { util } from '@chainlink/ea-bootstrap'
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig } from '@chainlink/types'
+
+export const NAME = 'dominance'
+
+const dominanceParams = {
+  market: ['market', 'to', 'quote'],
+}
+
+export const execute: ExecuteWithConfig = async (request, config) => {
+  const validator = new Validator(request, dominanceParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+
+  const url = 'https://pro-api.coinmarketcap.com/v1/global-metrics/quotes/latest'
+
+  const headers = {
+    'X-CMC_PRO_API_KEY': util.getRandomRequiredEnv('API_KEY'),
+  }
```

Comment:
`makeConfig` function should set this configuration as `config.apiKey` and `config.api.baseUrl` that should be set to `DEFAULT_API_ENDPOINT` or `'https://pro-api.coinmarketcap.com/v1/'` if `API_ENDPOINT` is not set.